src/tezos/TezosHardwareWallet.ts

```diff
@@ -0,0 +1,22 @@
+import * as ledgerUtils from "../utils/LedgerUtils";
+import {HardwareDeviceType} from "../types/HardwareDeviceType";
+import {base58CheckEncode} from "../utils/CryptoUtils";
+import * as sodium  from 'libsodium-wrappers-sumo';
+import {KeyStore, StoreType} from "../types/KeyStore";
+
+export namespace TezosHardwareWallet {
+
+    export async function unlockAddress(deviceType: HardwareDeviceType, derivationPath: string): Promise<KeyStore> {
+        const hexEncodedPublicKey = await ledgerUtils.getTezosPublicKey(derivationPath);
+        const publicKeyBytes = sodium.from_hex(hexEncodedPublicKey).slice(1);
```

Comment:
Should we add a clarifying comment as to why we're slicing a byte off here, for documentation?

Reply:
Yes. :smile: